cli/src/commands/describe.rs

```diff
@@ -67,6 +68,15 @@ pub(crate) struct DescribeArgs {
         conflicts_with = "stdin"
     )]
     message_paragraphs: Vec<String>,
+    /// Add a footer entry
+    ///
+    /// The value is a commit template. The output generated with that template
+    /// must be a valid footer entry.
```

Comment:
If the trailer is a template, it means that the user will have to quote string values, which then needs to be shell-quoted to. That may be hard for the user to guess.